addons/account/models/account_move.py

```diff
@@ -3661,6 +3661,15 @@ class AccountMoveLine(models.Model):
         context={'active_test': False},
         check_company=True,
         help="Taxes that apply on the base amount")
+    tax_move_line_ids = fields.Many2many(
+        comodel_name="account.move.line",
+        string="Related Tax lines",
+        compute="_compute_tax_move_line_ids",
+        store=True,
+        relation="base_tax_line_mapping",
+        column1="base_line_id",
+        column2="tax_line_id",
+        help="Technical field for the General Tax Report")
```

Comment:
Instead of making a brand new m2m table, you could simply create a new model <base_line, tax_line, base_amount, base_amount_currency, tax_amount, tax_amount_currency> then... it will solve all problems

Reply:
How do you suggest to fill that table? Override `write` in `account.move.line`?

Reply:
A computed one2many fied maybe ?